doc/architecture/computing-services.md

```diff
@@ -0,0 +1,105 @@
+
+Table of Contents
+=================
+
+   * [Table of Contents](#table-of-contents)
+   * [Motivation](#motivation)
+   * [Architecture](#architecture)
+      * [Enabling Publisher Services (Brizo)](#enabling-publisher-services-brizo)
+      * [Responsibilities](#responsibilities)
+      * [Initial solution](#initial-solution)
+      * [Next steps](#next-steps)
+
+
+---
+
+This page describes the highlights of the integration of additional computing services.
+
+# Motivation
+
+The most basic scenario of a Publisher, is to provide access to the Assets this Publisher owns or manage. In addition to this, having this data ecosystem in place,
+Publishers could offer other related services in top of this. Some possibles scenarios are:
+
+* The Publishers offer as a service the possibility of execute some computation on top of their data. This has some benefits:
+  - The data never leaves the Publisher enclave
+  - It's not necessary to move the data, the algorithm moves to the data
+  - Regulation
```

Comment:
"Regulation"? It's not clear what is meant by that one word here. How is it a benefit?